### PR TITLE
Add mtDNA Haplogroup by Ancestry FAQ

### PR DESCRIPTION
**Pull request by rileyhgrant**
Add FAQ question and respective answer per
https://the-tgg.slack.com/archives/C58J03659/p1656595760961199

I (believe) the snapshot test appears to be working as intended. All feedback welcome, of course.

### CODE REVIEW

**Comment by rileyhgrant** [comment]
Closing for now - didnt lint

**Comment by phildarnowsky-broad** [comment]
@rileyhgrant if everything was good except you just needed to lint what you wrote in the last commit, there's good ways to do that without having to open a fresh PR and all that rigamarole--in fact, I also forgot to lint a PR the other day, but was able to make a quick tweak and move on.

Related, git is a big and complex tool and it pays to spend a little time learning about its capabilities, this is a good resource: https://gitready.com/

Also /cc @mattsolo1 this makes me think we should some documentation on workflow, beginning with what process we would like to follow that takes us from the point where you have code ready to review but solely on your own machine, to the point where that code is in `main`. I was able to make some educated guesses that seem to have worked out, but some more explicit guidance would be helpful for Riley.

**Comment by rileyhgrant** [comment]
@phildarnowsky-broad, yeah I'll definitely review a bit more. I'm not totally new to git, but in this instance I got scared off by the CI/CD checks failing, so decided to close it then open it again once they were passing - to (hopefully) not waste anyone's time in reviewing a PR that wasn't passing automated tests anyhoo.

**Review by mattsolo1 — APPROVED** [review]

File: browser/help/faq/mitochondrial-dna/how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population.js
Comment (phildarnowsky-broad) [inline browser/help/faq/mitochondrial-dna/how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population.js:98]: The snapshot test is a good start here, but we also have new interactions on this page that should be covered as well, i.e. the data updating when you change your selection in these `select`s. However, that would require some more setup and possibly some choices between different tools, which would expand the scope of this PR a bit more than we might want, while conversely it'd fold naturally into the work on test infrastructure I've been doing lately.

So what I think is, let's leave test that out of this PR, but after it's merged I will come back and use this interaction as a testbed for this other style of test. Then the test for this interaction can serve as an example for similar tests going forward.

File: browser/help/faq/mitochondrial-dna/how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population.js
Comment (rileyhgrant) [reply browser/help/faq/mitochondrial-dna/how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population.js:30]: This makes sense on explicit identifiers, I'll continue to be more explicit going forward.

File: browser/help/faq/mitochondrial-dna/how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population.js
Comment (rileyhgrant) [reply browser/help/faq/mitochondrial-dna/how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population.js:84]: I like the rule of thumb of 3 times, good to know. Thanks for the suggestions and well formatted information!

File: browser/help/faq/mitochondrial-dna/how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population.js
Comment (rileyhgrant) [reply browser/help/faq/mitochondrial-dna/how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population.js:98]: This sounds good to me.

File: browser/help/helpPageTableOfContents.json
Comment (phildarnowsky-broad) [reply browser/help/helpPageTableOfContents.json:97]: interesting, yes let's dig into that tomorrow, not worth blocking the PR I would say

File: browser/help/faq/mitochondrial-dna/how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population.js
Comment (phildarnowsky-broad) [inline browser/help/faq/mitochondrial-dna/how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population.js:178]: nice catch on the name, that's the idea

**Comment by mattsolo1** [comment]
@phildarnowsky-broad how is this looking? Are there still changes Riley needs to make?

If we're good, what is the right thing for Riley to do now?

Given Phil's comment on Slack:

> ideal end result is that the git history looks like you got the code right the first time, but not necessarily that 1 PR = 1 commit. on getting feedback, I typically would make multiple small commits with one fix apiece, then use an interactive rebase to reorder commits so each fix comes after an appropriate commit, then once that's good, one more rebase to fold the fixes into the appropriate commits

Should Riley now fold everything into a single commit & force push again? Am I right in thinking we don't want commits titled something like "addressed reviewer suggestions"?

**Comment by mattsolo1** [comment]
Yeah, good idea to re-request a review when you're ready for another look.

**Review by phildarnowsky-broad — APPROVED** [review]
Congratulations on your upcoming first merge as a professional!

**Comment by phildarnowsky-broad** [comment]
@mattsolo1 @rileyhgrant I think too it would be good if we stick to regularly using Github's interface for requesting and providing review, since that way it's clear-cut when things are ready for review, re-review, or merge, and it saves time and attention if all you have to do is hit a single button.

**Comment by rileyhgrant** [comment]
> @mattsolo1 @rileyhgrant I think too it would be good if we stick to regularly using Github's interface for requesting and providing review, since that way it's clear-cut when things are ready for review, re-review, or merge, and it saves time and attention if all you have to do is hit a single button.

Roger that. Thanks both for getting up me up to speed on better Git and Github practices - it's much appreciated.

**Pull request merged**
